doc/SRS.md

```diff
@@ -0,0 +1,174 @@
+
+### 3.3 Reliability
+#### 3.3.1 Availability
+While we cannot 100% guarantee it, we want to ensure that the server is available at least 95% of the time, equivalent to approx. 1 hour of downtime per day.
+The time to repair bugs etc. should be as low as possible.
+#### 3.3.2 Defect Rate and Security
+Data should be stored in the most secure way possible. The data transfer is to be made via a secure connection.
+
+### 3.4 Performance
+#### 3.4.1 Response time
+Should be as low as possible. Maximum response time is 5 seconds.
+#### 3.4.2 Capacity
+The system should be able to manage multiple requests per second without noticeable latency. The system is to be build on a scalable basis.
+#### 3.4.3 Connection bandwidth
+The size of data received through one request should be as low as possible, e.g. sending duplicates of data should be avoided.
+
+### 3.5 Supportability
+#### 3.5.1 Coding standards
+In order to maintain supportability and readability of our code, we will adopt the clean code standard, common naming conventions uniform formatting and best practices throughout the project.
+#### 3.5.2 Maintenance Utilities
+In order to test and integrate newer versions of the application, a continuous integration service is required.
+
+### 3.6 Design Constraints
+Our goal is to provide a modern design in both code and application.  
+Server-side programs ought to be compiled using Golang. A REST-API will be used to communicate between client and server.  
+The dashboard will be implemented using a MVC architecture.  
+The chrome-plugin and the android autofill service will be implemented using their respective technologies and languages.  
+#### 3.6.1 Development tools
+- Version control system: Git (Github)
+- Backend development: JetBrains GoLand
+- Frontend development: tbd
+- Project planning tool: JetBrains YouTrack
+- Build management: tbd
+- CI: tbd
+
+### 3.7 On-line User Documentation and Help System Requirements
+As stated our goal is to make our application as intuitive as possible. However, we will provide a FAQ and documentation. This will be especially helpful for users who want to know how our application works in depth.
+
+### 3.8 Purchased Components
+A server hosted at [zap-hosting.com](https://zap-hosting.com/de/) is used to run the frontend application as well as the backend server.  
+Currently there are no other purchased components.
+
+### 3.9 Interfaces
+#### 3.9.1 User Interfaces
+There will be following interfaces available:
+- Dashboard: manage (add, remove), retrieve passwords, change settings
+- Chrome-Plugin: manage, retrieve passwords
+#### 3.9.2 Hardware Interfaces
+n/a
+#### 3.9.3 Software Interfaces
+As the dashboard is a web-application it should be running on the most common browsers. Nevertheless, we want to primarily support the Google Chrome browser and its features.
+#### 3.9.4 Communications Interfaces
+The client will connect to the server `https`. An unencrypted connection wil not be supported. 
```

Comment:
- "The client-server communication uses the encrypted `HTTPS` protocol. Unencrypted connections are not supported"